manifests/exporter.pp

```diff
@@ -5,13 +5,28 @@
 #   include rsan::exporter
 class rsan::exporter {
 
-
-
 ########################1.  Export Logging Function######################
 # Need to determine automatically the Network Fact IP for the RSAN::importer node automatically, applies to all infrastructure nodes
 #########################################################################
 
-
+class { '::nfs':
+  server_enabled => true
+  }
+  nfs::server::export{ '/var/log/':
+  ensure  => 'mounted',
```

Comment:
These lines should be indented 2 more than the parent since we're inside a set of curly braces i.e. 

```suggestion
    ensure  => 'mounted',
```